ibis/backends/flink/__init__.py

```diff
@@ -400,7 +496,44 @@ def drop_table(
             Name of the database where the table exists, if not the default.
         catalog
             Name of the catalog where the table exists, if not the default.
-        temp
+        force
+            If `False`, an exception is raised if the table does not exist.
+        """
+        self._drop_table(
+            name=name,
+            database=database,
+            catalog=catalog,
+            temporary=True,
+            force=force,
+        )
+        self._drop_table(
+            name=name,
+            database=database,
+            catalog=catalog,
+            temporary=False,
+            force=force,
+        )
```

Comment:
If there are two to make sure the view gets dropped whether it is `temporary` or not, I would suggest updating `_drop_table` so you can pass in an argument to that effect, instead of duplicating the call with a single argument changed.

Reply:
Removed while bringing `temp` back.